client/src/app/site/agenda/components/item-info-dialog/item-info-dialog.component.html

```diff
@@ -0,0 +1,28 @@
+<h1 mat-dialog-title>Change values for {{ item.getTitle() }}</h1>
+<div mat-dialog-content>
+    <form class="itemDialogForm" [formGroup]="agendaInfoForm" (keydown)="onKeyDown($event)" (ngSubmit)="saveItemInfo()">
+        <mat-form-field>
+            <mat-select formControlName="type" placeholder="{{ 'Agenda visibility' | translate }}">
+                <mat-option *ngFor="let type of itemVisibility" [value]="type.key">
+                    <span>{{ type.name | translate }}</span>
+                </mat-option>
+            </mat-select>
+        </mat-form-field>
+
```

Comment:
A checkbox for `done` (internally named `closed`) woudl be nice incl. a visual feedback in the list view if the item is done.